hbase-server/src/main/java/org/apache/hadoop/hbase/mob/MobConstants.java

```diff
@@ -55,33 +55,22 @@
   public static final long DEFAULT_MOB_CACHE_EVICT_PERIOD = 3600L;
 
   public final static String TEMP_DIR_NAME = ".tmp";
-  public final static String BULKLOAD_DIR_NAME = ".bulkload";
-  public final static byte[] MOB_TABLE_LOCK_SUFFIX = Bytes.toBytes(".mobLock");
-  public final static String EMPTY_STRING = "";
-  /**
-   * If the size of a mob file is less than this value, it's regarded as a small file and needs to
-   * be merged in mob compaction. The default value is 1280MB.
-   */
-  public static final String MOB_COMPACTION_MERGEABLE_THRESHOLD =
-    "hbase.mob.compaction.mergeable.threshold";
-  public static final long DEFAULT_MOB_COMPACTION_MERGEABLE_THRESHOLD = 10 * 128 * 1024 * 1024;
+
   /**
-   * The max number of del files that is allowed in the mob file compaction. In the mob
-   * compaction, when the number of existing del files is larger than this value, they are merged
-   * until number of del files is not larger this value. The default value is 3.
+   * The max number of a MOB table regions that is allowed in a batch of the mob compaction.
+   * By setting this number to a custom value, users can control the overall effect
+   * of a major compaction of a large MOB-enabled table.
    */
-  public static final String MOB_DELFILE_MAX_COUNT = "hbase.mob.delfile.max.count";
```

Comment:
for these configs that are no longer used, on branches where we can't remove them yet we should emit a warning if they are present in configuration.

Reply:
MobFileCleanerChore checks old configs and log warnings already. I will restore all removed old configs in MobConstants.